bedrock/newsletter/forms.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 # This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Has the `EmailField` line above been deleted deliberately?

Reply:
You might need to see the diff in its full view to see what i mean here - the  line above the deleted line has poss also been deleted (or the diff rendering implied that)